src/sea.h

```diff
@@ -1,8 +1,7 @@
-#pragma once
-#include <stdio.h>
+#ifndef _SEA_H
```

Comment:
Yeah, that would make sense, in general. FWIW `#pragma once` is supported across a fair number of very common compiler chains: https://en.wikipedia.org/wiki/Pragma_once#Portability